lib/TaffoUtils/Metadata.h

```diff
@@ -41,6 +41,7 @@
 #define ORIGINAL_FUN_METADATA  "taffo.originalCall"
 #define CLONED_FUN_METADATA    "taffo.equivalentChild"
 #define SOURCE_FUN_METADATA    "taffo.sourceFunction"
+#define INDIRECT_METADATA      "taffo.indirectFunction"
```

Comment:
Nothing against this specific change but...are you sure it is in the right Pull Request?

Reply:
I thought about "feature" branches so we called all of them openmp, but this looks a bit out of place indeed so I'm going to do some rebase and create a separate branch/PR for the Metadata changes 🆗

Reply:
Moved the metadata commits here https://github.com/HEAPLab/TAFFO/pull/5